spec/features/navigation/guest_user_can_visit_home_page_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.feature "user can visit home page" do
+
+  before :each do
+    create(:project)
+  end
+
   context "as a guest user" do
```

Comment:
from `app/views/home/index.html.erb`

```erb
<div class="container row">

	<div class="col s4">
  <div class="card medium m3 l3">
  	<h5>Trending</h5>
    <% project = @projects.order("title asc").first %>
    <div class="card-image waves-effect waves-block waves-light">
      <%= image_tag project.image_url, class: "activator responsive-img" %>
    </div>
```

The problem was calling `project.image_url`